website/docs/core-concepts/writing-code/javascript-promises.md

```diff
@@ -38,147 +35,129 @@ However, using Promise for the same problem makes the implementation easier and
  }}
 ```
 
-### Async/Await
-
-The `async` and `await` keywords enable the [asynchronous](javascript-editor-beta/#asynchronous) workflow to be written in a cleaner style, avoiding the need to configure promise chains explicitly.
-
-#### Async
-
-Adding the `async` keyword before a function means one simple thing: a function always returns a promise. Other values are wrapped in a resolved promise automatically.
-
-#### Await
+## Async/Await
+The `async` and `await` keywords enable the [asynchronous](/core-concepts/writing-code/javascript-editor-beta#asynchronous) workflow to be written in a cleaner style, avoiding the need to configure promise chains explicitly.
 
-The keyword `await` makes JavaScript wait until that Promise settles and returns its result.
+### Async
+Adding the `async` keyword before a function always returns a promise. Other values are wrapped in a resolved promise automatically.
 
-Let's take an example to understand these keywords. Refer to the code below:
+### Await
+The keyword `await` makes JavaScript wait until that Promise settles and returns its result. The below example code shows the use of `async` and `await` keywords:
 
-```
-{{ 
-(async function(){ 
-    const response = await MockApi.run({ name: 'Appsmith' }); 
-    await storeValue( "name", response.args.name ); 
-    await showAlert(appsmith.store.name); 
-    })() 
+```javascript
+{{
+	(async function(){ 
+		const response = await MockApi.run({ name: 'Appsmith' }); 
+		await storeValue( "name", response.args.name ); 
+		await showAlert(appsmith.store.name); 
+	})() 
 }}
 ```
+The code flow shows that:
+1. You run `MockApi` with the parameter name as "Appsmith" and wait for the response.
+2. You store the response in the Appsmith store using `storeValue` when you get the response.
+3. On completion of `storeValue`, you show an alert with a message.
 
-In the example above:
-
-1. We run `MockApi` with the parameter name as "Appsmith" and wait for the response.
-2. We store the response in the Appsmith store using `storeValue` when we get the response.
-3. On completion of `storeValue`, we show an alert with a message.
 
-### Promise methods
-
-If you want only one action/promise to finish for further execution, you can use `Promise.any()` or `Promise.race()` methods.
```

Comment:
Let's keep this line

Reply:
It is added to the individual function refs for .`any()` and `.race()`